crates/uv/src/commands/project/init.rs

```diff
@@ -571,6 +608,102 @@ impl InitProjectKind {
 
         Ok(())
     }
+
+    #[allow(clippy::fn_params_excessive_bools)]
+    async fn init_script(
+        self,
+        script_path: &PathBuf,
+        python: Option<String>,
+        connectivity: Connectivity,
+        python_preference: PythonPreference,
+        python_downloads: PythonDownloads,
+        cache: &Cache,
+        printer: Printer,
+        no_workspace: bool,
+        no_readme: bool,
+        no_pin_python: bool,
+        package: bool,
+        native_tls: bool,
+    ) -> Result<()> {
+        if no_workspace {
+            warn_user_once!("`--no_workspace` is a no-op for Python scripts, which are standalone");
+        }
+        if no_readme {
+            warn_user_once!("`--no_readme` is a no-op for Python scripts, which are standalone");
+        }
+        if package {
+            warn_user_once!("`--package` is a no-op for Python scripts, which are standalone");
+        }
+        let client_builder = BaseClientBuilder::new()
+            .connectivity(connectivity)
+            .native_tls(native_tls);
+
+        let reporter = PythonDownloadReporter::single(printer);
+
+        if script_path.try_exists()? {
+            anyhow::bail!("Script already exists at {}", script_path.to_str().unwrap());
```

Comment:
I sort of think it could be okay to accept an existing file. And if the file exists, and doesn't contain a PEP 723 script tag, we add it, but otherwise don't modify the script.

Reply:
That makes sense, implemented this + modified the test to reflect the new behavior